### DISCUSSION TITLE
Circumcision

### DISCUSSION THREAD

**Discussion by HanKortekaas** [FamilySearch/GEDCOM#138] [Ideas]
we are anabling the gedcom-7 dialect into our software and would want to install the possibility for jewish and muslem believers to register circumstancing. What Tag / structure would you recommend since I cannot find it in the technical description.

**Comment by tychonievich** [comment]
I'm not familiar with "circumstancing"; do you mean "circumcision"?
You are correct that we do not have an event tag for that event type. Issue #117 tracks progress on a proposal to add this along with many more missing event types.

**Reply by HanKortekaas** [reply]
Sorry, I meant Circumcision
